Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationImportedPfxCertificatePolicyWindows10/MSFT_IntuneDeviceConfigurationImportedPfxCertificatePolicyWindows10.schema.mof

```diff
@@ -0,0 +1,32 @@
+[ClassVersion("1.0.0.0")]
+class MSFT_DeviceManagementConfigurationPolicyAssignments
```

Comment:
This name will create conflicts with the previous PR I merged. As discussed previously, let name the CIMInstances with

MSFT_resourceNameCimInstance

In this case this would be MSFT_IntuneDeviceConfiguratonImportPfxCertificatePolicyWindows10Assignments, or something similar. Please review the other PRs as well to ensure we prevent any naming conflicts. Thanks